src/metabase/pulse.clj

```diff
@@ -291,8 +291,8 @@
           retry-config (retry/retry-configuration)
           retry-errors (volatile! [])
           retry-report (fn []
-                         {:attempted-retries (count @retry-errors)
-                          :retry-errors       @retry-errors})
+                         {:attempted_retries (count @retry-errors)
```

Comment:
This was added recently in 51.

these keys are stored in db so I think it's better to use underscore, if we're ok with this I'll create a PR to backport this part to 51.

Reply:
Makes sense IMO